frontend/src/pages/ArticleDetails.js

```diff
@@ -14,12 +14,13 @@ function ArticleDetails() {
 
     const fetchArticleDetails = async () => {
       try {
-        const result = await apiCallWithAuth(`/api/article/${article_id}/`, 'GET');
+        const result = await axios.get(`/api/article/${article_id}/`);
 
         if (isMounted) { // Only update state if component is still mounted
-          if (result.success) {
-            result.data.content_paragraphs = result.data.content_paragraphs.filter(paragraph => paragraph.trim() !== '');
-            setArticleDetails(result.data);
+          if (result.status === 200) {
+            const data = result.data;
+            data.content_paragraphs = data.content_paragraphs.filter(paragraph => paragraph.trim() !== '');
+            setArticleDetails(data);
           } else {
             setError('Failed to fetch article details');
           }
```

Comment:
Same question here. Calling API in `try` block will always move to `catch` when the API returns non-success status codes.